js/classes/entities/foreground/hud.js

```diff
@@ -22,6 +23,37 @@ class Hud {
         return 10;
     }
 
+    swapToCrosshair() {
+        let crosshairCursorUnclicked = 'url(../sprites/crosshair_unpressed.png) 16 16, auto';
+        let crosshairCursorClicked = 'url(../sprites/crosshair_pressed.png) 16 16, auto';
```

Comment:
It might make sense to rename the PNGs for the crosshair such that the naming of the variables and related PNGs are consistent. Like crosshair_clicked.png etc.

Reply:
Whoops! Good catch. I'll make the naming consistent.